src/Sylius/Bundle/CoreBundle/spec/Sitemap/Model/SitemapUrlSpec.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace spec\Sylius\Bundle\CoreBundle\Sitemap\Model;
+ 
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\ChangeFrequency;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapUrlInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapUrlSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapUrl');
+    }
+
+    function it_implements_sitemap_url_interface()
+    {
+        $this->shouldImplement(SitemapUrlInterface::class);
+    }
+
+    function it_has_localization()
+    {
+        $this->setLocalization('http://sylius.org/');
+        $this->getLocalization()->shouldReturn('http://sylius.org/');
+    }
+
+    function it_has_last_modification(\DateTime $now)
+    {
+        $this->setLastModification($now);
+        $this->getLastModification()->shouldReturn($now);
+    }
+
+    function it_has_change_frequency()
+    {
+        $changeFrequency = ChangeFrequency::always();
```

Comment:
That's kind of hack as we are comparing the references.

I'm not sure that it's the best idea, but casting `ChangeFrequency` to `string` inside `getChangeFrequency()` may work.